src/test/java/org/openmrs/contrib/qaframework/page/ConditionsPage.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.contrib.qaframework.page;
+
+import java.util.List;
+
+import org.openmrs.contrib.qaframework.helper.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class ConditionsPage extends Page {
+
+	public static final By ADD_NEW_CONDITION = By.id("conditionui-addNewCondition");
+	public static final By SET_ACTIVE = By.xpath("//button[text()='Set Active']");
+	public static final By SET_INACTIVE = By.xpath("//button[text()='Set Inactive']");
+	private static final By RETURN = By.cssSelector(".actions .cancel");
+	private static final By TAB_ACTIVE = By.cssSelector("a[href='#ACTIVE']");
+	private static final By TAB_INACTIVE = By.cssSelector("a[href='#INACTIVE']");
+	private static final By FIRST_CONDITION_NAME = By.xpath("//table/tbody[2]/tr[1]/td[1]");
+	private static final By EDIT = By.cssSelector("i[title='Edit Condition: ']");
+	private static final By DELETE = By.cssSelector("i[title='Delete']");
+	private static final By CONDITIONS_LIST = By.cssSelector("tr.clickable-tr");
+	private static final By CONFIRM_DELETE_CONDITION_BUTTON = By.cssSelector("#remove-condition-dialog > div.dialog-content > button.confirm.right");
+
+	public ConditionsPage(ClinicianFacingPatientDashboardPage clinicianFacingPatientDashboardPage) {
+		super(clinicianFacingPatientDashboardPage);
+	}
+
+	public ConditionsPage(ConditionPage conditionPage) {
+		super(conditionPage);
+	}
+
+	@Override
+	public String getPageUrl() {
+		return "/coreapps/conditionlist/manageConditions.page";
+	}
+
+	public ClinicianFacingPatientDashboardPage clickReturn() {
+		clickOn(RETURN);
+		return new ClinicianFacingPatientDashboardPage(this);
+	}
+
+	public String getFirstConditionName() {
+		try {
+			return driver.findElement(FIRST_CONDITION_NAME).getAttribute(
```

Comment:
hey @sherrif10 , the changes by [this Pull request](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/582) are not captured here-->  https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/src/test/java/org/openmrs/contrib/qaframework/page/ConditionsPage.java ..

if am getting this right, you're  bringing all the pages from distro into qa. Enlighten me if am lost

Reply:
Add the ticket for that fix in qaframework module

Reply:
Simply create a new ticket in qaframework module copy and paste only the new code which was introduced in that ticket

Reply:
please checkout -> https://github.com/openmrs/openmrs-contrib-qaframework/pull/171